tapir/statistics/views.py

```diff
@@ -206,6 +217,62 @@ def get_context_data(self, **kwargs):
         return context_data
 
 
+class AverageBasketEvolutionJsonView(LoginRequiredMixin, JSONView):
+    def get_context_data(self, **kwargs):
+        tapir_user = get_object_or_404(TapirUser, pk=(self.kwargs["pk"]))
+        logged_user = self.request.user
+        if tapir_user.pk != logged_user.pk and not logged_user.has_perm(
+            PERMISSION_COOP_MANAGE
+        ):
+            return HttpResponseForbidden("Access forbidden")
+
+        user_purchases = (
+            PurchaseBasket.objects.filter(tapir_user=tapir_user)
+            .annotate(month=TruncMonth("purchase_date"))
+            .values("month")
+            .annotate(average_gross_amount=Avg("gross_amount"))
+            .order_by("month")
+        )
+
+        months = self.get_months(user_purchases)
+
+        return build_line_chart_data(
+            x_axis_values=months,
+            y_axis_values=[self.get_avg_price_per_month(user_purchases, months)],
+            data_labels=[_("Price of average basket")],
+        )
+
+    @staticmethod
+    def get_months(user_purchases):
+        months = []
+        if len(user_purchases) > 0:
+            now = datetime.datetime.now().replace(
+                day=1, hour=0, minute=0, second=0, microsecond=0, tzinfo=utc
+            )
+
+            month = user_purchases[0]["month"].replace(tzinfo=utc)
+            months.append(month.strftime("%Y-%m"))
+
+            while month != now:
+                month = month + relativedelta(months=1)
```

Comment:
You can use `tapir.utils.shortcuts.get_first_of_next_month` to avoid relying on the `dateutil` dependency, we don't use it explicitely yet (I think it's only there because it's a dependency of celery).